tests/code/test_users.py

```diff
@@ -322,3 +321,51 @@ def test_getUserBGCheckHistory():
 
 
         transaction.rollback()
+
+@pytest.mark.integration
+def test_getBannedUsers():
+    with mainDB.atomic() as transaction:
+        bannedUser = User.create(username = 'usrtst', # Test banned user
```

Comment:
I think the name should not be bannedUser because at this point the user has not been banned yet.

Reply:
Changed to userToBan